test/integration/lib/helper/configurations.js

```diff
@@ -23,7 +23,7 @@ test('findBoletoConfiguration: without configuration created', async (t) => {
   payload.external_id = externalId
   payload.issuer = 'boleto-api-bradesco-shopfacil'
 
-  const boleto = await findBoletoConfiguration(payload)
+  const boleto = await findBoletoConfiguration(payload, 'operationId')
```

Comment:
Sugestão super opcional pra os testes mais parecidos com o log real 
```suggestion
  const operationId = cuid()
  const boleto = await findBoletoConfiguration(payload, operationId)
```

Reply:
feito!